strawberry/fastapi/__init__.py

```diff
@@ -0,0 +1,129 @@
+from typing import Any, Dict, Optional
+
+from pydantic import Field
+
+from fastapi import APIRouter, Body, HTTPException
+from fastapi.responses import HTMLResponse
+from fastapi.websockets import WebSocket
+from strawberry.asgi import GraphQL
+from strawberry.asgi.utils import get_graphiql_html
+from strawberry.http import GraphQLHTTPResponse, GraphQLRequestData, process_result
+from strawberry.utils.debug import pretty_print_graphql_operation
+
+
+NO_GRAPHIQL_MSG = "GraphiQL not enabled"
+GRAPHIQL_EXAMPLE = """
```

Comment:
can we move the these top level variables to different files?

Reply:
Where would you like them? :)

Reply:
mmh

`GraphQLRouter` -> `fastapi/router.py`
`NO_GRAPHIQL_MSG` and `GRAPHIQL_EXAMPLE` -> `fastapi/constants.py`

Reply:
You bet!